hal/armv7a/_interrupts.S

```diff
@@ -0,0 +1,281 @@
+/*
+ * Phoenix-RTOS
+ *
+ * Operating system kernel
+ *
+ * Interrupts handlers for Cortex-A9
+ *
+ * Copyright 2018, 2020, 2021 Phoenix Systems
+ * Author: Pawel Pisarczyk, Aleksander Kaminski, Maciej Purski, Hubert Buczynski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
```

Comment:
We might need need to make sure that this file is compiled to ARM asm.

Reply:
Done